frontend/tests/acceptance/gitlab-test.js

```diff
@@ -0,0 +1,220 @@
+import { module, test } from 'qunit'
+import { setupApplicationTest } from 'ember-qunit'
+import {
+  authenticateSession,
+  invalidateSession
+} from 'customer-center/tests/helpers/ember-simple-auth'
+import startApp from '../helpers/start-app'
+import destroyApp from '../helpers/destroy-app'
+import { selectChoose, clearSelected } from 'ember-power-select/test-support'
+import { clickTrigger } from 'ember-basic-dropdown/test-support/helpers'
+import { calendarSelect } from 'ember-power-calendar/test-support'
+import moment from 'moment'
+
+module('Acceptance | GitLab', function(hooks) {
+  setupApplicationTest(hooks)
+
+  hooks.beforeEach(function() {
+    this.application = null
+  })
+
+  hooks.beforeEach(async function() {
+    this.application = startApp()
+    let user = server.create('user', 'customer')
+    await authenticateSession(this.application, { data: user.id })
+  })
+
+  hooks.afterEach(async function() {
+    await invalidateSession(this.application)
+    destroyApp(this.application)
+  })
+
+  test('[T08] All projects and namespaces are rendered', async function(assert) {
+    await visit('/projects')
+
+    assert.equal(find('[data-test-group]').length, 3)
+    assert.equal(find('[data-test-project]').length, 12)
```

Comment:
`assert.dom('[data-test-project]').exists({ count: 12 })` same for groups

Reply:
:+1: